BackendWebApp/src/test/java/uk/ac/ebi/biostd/webapp/application/rest/controllers/PendingSubmissionResourceTest.java

```diff
@@ -205,11 +201,13 @@ public void testDeleteSubmission() throws Exception {
     public void testSubmitSubmission() throws Exception {
         final String accno = "1234";
 
-        when(pendingSubmissionService.submitSubmission(accno, user))
+        when(pendingSubmissionService.submitSubmission(eq(accno), any(JsonNode.class), eq(user)))
                 .thenReturn(Optional.of(
                         SubmitReportDto.builder().status(SubmitStatus.OK).build()));
 
-        mvc.perform(post("/submissions/pending/{accno}/submit", accno))
+        mvc.perform(post("/submissions/pending/{accno}/submit", accno)
+                .content("{}")
```

Comment:
I think is worth it to create constant  for `{}` Empty_object not because re using but because it helps to understant the meaning which can be confusing for one

Reply:
All done.